content/build/cloud/ci.md

```diff
@@ -290,6 +290,37 @@ pipeline {
   }
 }
 ```
+Using the `build.sh` shell script in your `.travis.yml` file under the `install` hook. In this example Travis CI is using Node: 
+
+{{< tabs >}}
+{{< tab name="Travis CI" >}}
+
+```yaml
+language: minimal 
+dist: jammy 
+
+services:
+  - docker
+
+env:
+  global:
+    - IMAGE_NAME=username/repo
+
+before_install: |
+  echo "$DOCKER_PASS" | docker login --username "$DOCKER_USER" --password-stdin
+  sudo apt-get update && sudo apt-get install -y jq build-essential
+  npm --version
+  npm config set loglevel verbose
+  npm config set progress false
+  npm cache clean --force
+
+install: |
+  set -e 
+  chmod +x ./build_cloud.sh; ./build_cloud.sh
```

Comment:
Hi @dvdksn, 

```yaml
language: minimal 
dist: jammy 

services:
  - docker

env:
  global:
    - IMAGE_NAME=username/repo

before_install: |
  echo "$DOCKER_PASS" | docker login --username "$DOCKER_USER" --password-stdin
install: |
  set -e 
  chmod +x ./build_cloud.sh
 ./build_cloud.sh

script: |
  docker buildx create --name username --use
``` 

Per your request, here is a minimal `.travis.yml` configuration that has been successfully executed and built. The inclusion of the `build-essential` package, for instance, is intentional to ensure the installation of its latest version. This acts as a precautionary measure, despite Travis CI's default virtual machine image typically having these packages pre-installed and depending on what `dist` one person may be using for their pipeline it may be different.